Jasmine/Common/ViewModels/TimedViewModelProtocol.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/// A ViewModel that implements a timer, i.e. it is timed.
+protocol TimedViewModelProtocol: BaseViewModelProtocol {
```

Comment:
Actually timer should only have the VM, not the VC. 
It will call VM methods which will in turn call VC methods.
I'm not sure if the VM is stored as a weak reference when you pass it as argument and get used in the .scheduledTimer closure?